templates/default/example.org.erb

```diff
@@ -0,0 +1,14 @@
+; Generated by Chef for <%= node['fqdn'] %>
+; Local modifications will be overwritten.
+
+$TTL 86400      ; 1 day
+$ORIGIN <%= @zone %>.
+@    IN     SOA  ns.example.net. <%= node['bind']['zones']['email'] %> (
```

Comment:
It makes zero sense to ship this template with hardcoded SOA/NS records for example.net in the default recipe of this cookbook.  See previous comments at https://github.com/atomic-penguin/cookbook-bind/pull/20/files#123.